security/ThreatModel.md

```diff
@@ -0,0 +1,11 @@
+Creating a website, like 2 People Prompt, that allows users to create posts adds additional security threats that should be address to lower the attack risks. Three of the biggest threats that comes with allowing users to write to the site are posting inapproriate content spamming and code injecting. Malicious users and bots could try to crash the server by submitting a post that overloads the storage what a server can handle or that we are allowed to use. They could try to inject code into the site to redirect users or to steal personal information. Hackers could also misuse the site by posting tromblesome content, including adult only material, illegal and copyrigthed items.
+
+To help eliminate or lower some of threats associated with allowing users to create posts some steps that can be take are:
+
+	- Use OAuth Service to help verify users
+	- require users to use a strong passphrase
+	- Limit the size that user can submit
+	- Santitize all content to be read only strings with links disabled
+	- Have clear rules on what is allowed and not allowed in post and have easy clear way for inappronate content to be reported
+	
```

Comment:
All great ideas for protecting your site! You should always assume that somebody is going to try and break your apps, so it's good that you've thought about these things.